tools/review/get_comments_markdown.py

```
@@ -0,0 +1,296 @@
+The script paginates over review comments and issue comments so very large PRs
+are fully captured. It does not filter by author.
+"""
+from __future__ import annotations
+
+import argparse
+import json
+import os
+import subprocess
+import sys
+import textwrap
+from typing import Dict, List, Tuple
```

Ruff (0.14.3)

`typing.Dict` is deprecated, use `dict` instead

(UP035)